revdb/revdb.gyp

```diff
@@ -488,6 +488,10 @@
 							# Error in ../../thirdparty/libsqlite/include/qry_dat.h
 							'-Werror=return-type',
 						],
+						'libraries':
+						[
+							'-lm',
+						],
 					},
```

Comment:
Hopefully not, the rule for `loadable_module` in android-settings should cover revzip I think. We probably have to test each external individually and see if they need any further settings applied though :-(